--- a/hypr/hyprland/input.lua
+++ b/hypr/hyprland/input.lua
@@ -2,7 +2,7 @@
 
 hl.config({
     input = {
-        kb_layout          = "us",
+        kb_layout          = vars.kbLayout,
         numlock_by_default = false,
         repeat_delay       = 250,
         repeat_rate        = 35,
--- a/hypr/variables.lua
+++ b/hypr/variables.lua
@@ -11,6 +11,7 @@
     editor                     = "codium",
     fileExplorer               = "thunar",
     audioSettings              = "pavucontrol",
+    kbLayout                = "us",
 
     -- Touchpad
     touchpadDisableTyping      = true,
