--- a/foot/foot.ini
+++ b/foot/foot.ini
@@ -1,6 +1,6 @@
 shell=fish
 title=foot
-font=JetBrains Mono Nerd Font:size=12
+font=JetBrainsMono Nerd Font:size=12
 letter-spacing=0
 dpi-aware=no
 pad=25x25
--- a/hypr/hyprland/group.lua
+++ b/hypr/hyprland/group.lua
@@ -10,7 +10,7 @@
             border_locked_inactive = vars.inactiveWindowBorderColour,
         },
         groupbar = {
-            font_family               = "JetBrains Mono NF",
+            font_family               = "JetBrainsMono NF",
             font_size                 = 15,
             gradients                 = true,
             gradient_round_only_edges = false,
